runelite-client/src/main/java/net/runelite/client/plugins/agility/AgilityPlugin.java

```diff
@@ -115,6 +120,9 @@
 	private int lastAgilityXp;
 	private WorldPoint lastArenaTicketPosition;
 
+	@Getter
+	private int agilityLevel = 0;
```

Comment:
`int` fields initialize at 0 already, no need to explicitly define it.
```suggestion
	private int agilityLevel;
```